src/components/DpDataTable/DpDataTable.vue

```diff
@@ -381,7 +382,7 @@ export default {
         headerSelectHint: Translator.trans('aria.select.all'),
         lockedForSelection: Translator.trans('item.lockedForSelection'),
         searchNoResults: (searchTerm) => Translator.trans('search.no.results', { searchterm: searchTerm }),
-        tableLoadingData: Translator.trans('loading.data'),
+        tableLoadingData: de.loadingData,
```

Comment:
I am not sure if I like accessing the transkeys directly.
If we do so now, we have to find all transkeys the moment we want to internationalize the lib.
I suggest to use something like `trans('loadingData')`

and the trans-method knows about the translation-file and adds the lang-flag

with that I hope it will be easier to implement vue-i18n later since it would be easier to find (and may replace) `trans(.*?)`

Reply:
It's a good point, but I think it might be more work to create a trans() function now than to simply remove the `de.` part later. Searching for 'de.' should give you all the trans keys, so I don't think it will require a lot of effort.